docs/src/app/components/pages/components/buttons.jsx

```diff
@@ -58,9 +58,9 @@ export default class ButtonPage extends React.Component {
             desc: 'Override the inline hover color of the button\'s root element.',
           },
           {
-            name: 'label or children',
-            type: 'string (label) or HTML/React elements (children)',
-            header: 'required',
+            name: 'label',
+            type: 'string',
+            header: 'optional',
```

Comment:
Although this change makes sense to me, I'm not sure if it is the best idea.. what do you think @oliviertassinari ?